tests/e2e/tests/generate/component/component-duplicate.ts

```diff
@@ -1,24 +1,23 @@
-import * as path from 'path';
 import { ng } from '../../../utils/process';
 import { oneLine } from 'common-tags';
 
 export default function () {
   return ng('generate', 'component', 'test-component')
     .then((output) => {
-      if (!output.stdout.match(/update src[\\|\/]app[\\|\/]app.module.ts/)) {
+      if (!output.stdout.match(/UPDATE src[\\|\/]app[\\|\/]app.module.ts/)) {
         throw new Error(oneLine`
           Expected to match
-          "update src${path.sep}app${path.sep}app.module.ts"
-          in ${output}.`);
+          "UPDATE src/app/app.module.ts"
+          in ${output.stdout}.`);
       }
     })
     .then(() => ng('generate', 'component', 'test-component'))
     .then((output) => {
-      if (!output.stdout.match(/identical src[\\|\/]app[\\|\/]app.module.ts/)) {
+      if (!output.stdout.match(/ERROR! src[\\|\/]app[\\|\/]test-component[\\|\/]test-component.component.ts already exists./)) {
```

Comment:
The merge strategy is being passed in via the CLI... the only change to force (pun intended) this behavior is to change `--force`'s default to be true.